index.html

```diff
@@ -1,51 +1,56 @@
 <!doctype html>
-
 <html lang="en">
 <head>
   <meta charset="utf-8">
-
   <title>Sprint Challenge - Home</title>
-
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
-
 </head>
 
 <body>
     <div class="container">
-   
+ <header>    
+     <img src="img/lambda-black.png" alt="lambdalogo">
+    <a href="index.html">Home</a>
+    <a href="about.html">About</a>
+    <a href="">Products</a>
+    <a href="">Blog</a>
+    <a href="">Contact</a>
+</header>
+
+    <div class="container">
+
+
         <section class="top-content">
+
+
+
+            <img src="img/jumbo.jpg" alt="jumbo">
             <div class="text-container">
                 <h2>The Future</h2>
                 <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
-            </div>
-            <div class="text-container">
+            
                 <h2>The Past</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
-            </div>
-        </section>
-        
-        <section class="middle-content">
-        
+                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus</p>
+                </div>
+            </section>
             <h2>Why Did It Have To Be Boxes...</h2>
-            
-            <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
-            </div>
-        
-        </section>
 
-        <section class="bottom-content">
+            <ul> 
+                <li class=box1>Box 1</li>
+                <li class=box2>Box 2</li>
+                <li class=box3>Box 3</li>
+                <li class=box4>Box 4</li>
+                <li class=box5>Box 5</li>
+                <li class=box6>Box 6</li>
+                <li class=box7>Box 7</li>
+                <li class=box8>Box 8</li>
+                <li class=box9>Box 9</li>
+                <li class=box10>Box 10</li>
+            </ul>
```

Comment:
Great job, alternative solution  would be to use the nth-child selector so you do not have to modify the css